.idea/codeStyles/Project.xml

```diff
@@ -1,12 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
```

Comment:
I would remove this file from the repository, it was added by mistake:

`git rm .idea/codeStyles/Project.xml`

should do it :)